src/args/make-python-pypi-environment/default.nix

```diff
@@ -21,9 +21,12 @@
   withCython_0_29_24 ? false,
   withNumpy_1_24_0 ? false,
   withSetuptools_57_4_0 ? false,
```

Comment:
Let's get rid of older versions and only support latest

Reply:
Done!